src/Data/Algorand/Transaction/Signed.hs

```diff
@@ -256,3 +257,52 @@ instance ToJSON LogicSignature where
 
 instance FromJSON LogicSignature where
   parseJSON = parseCanonicalJson
+
+data BlockTransaction = BlockTransaction
+  { btSig :: TransactionSignature
+  , btTxn :: Transaction
+  , btHgh :: Bool
+  -- ^ [btHgh] whether the tx has genesis hash included in serialized representation.
+  , btHgi :: Bool
+  -- ^ [btHgi] whether the tx has genesis id included in serialized representation.
+  -- , btApplyData :: ApplyData
```

Comment:
Is it supposed to be uncommented later?

Reply:
maybe)
https://github.com/algorand/go-algorand/blob/916154b5088e25472f68cc7f2971b63176a3889d/data/transactions/transaction.go#L102